crates/swc_css_minifier/tests/fixture/compress-calc/ignore-selectors/input.css

```diff
@@ -1,5 +1,3 @@
 div[data-size="calc(3*3)"] {
-}
-
-div:nth-child(2n + calc(3*3)) {
```

Comment:
`calc` can't be used here, it is not allowed by spec here, so I remove this tests, because it is list of component values and we even parser this as `Calc`